### DISCUSSION TITLE
Multisite forms?

### DISCUSSION THREAD

**Discussion by riasvdv** [solspace/craft-freeform#10] [Feature Requests]
Is it possible to create forms that are only available on one site/locale?

**Comment by kjmartens** [comment]
This currently isn't available with Freeform, but I will add it to the feature requests list. :)

**Comment by krrrli** [comment]
Hi, 
Is there any news about forms that are only available on one site? 
Thanks

**Reply by kjmartens** [reply]
Sorry, there is no update on this one. We still plan to have support for this some day, though. 🙂

**Reply by krrrli** [reply]
Ok Thanks anyways! 🙂

**Comment by kjmartens** [comment]
We have a feature in the works that will essentially simulate this. Watch for Freeform 5.2 sometime next week. 🙂

**Comment by kjmartens** [comment]
Freeform 5.2 is now available and includes the ability to filter forms and submissions by Sites they are assigned to. 🙂 
https://docs.solspace.com/craft/freeform/v5/configuration/sites/
